elasticdl/python/tests/worker_test.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2020 The ElasticDL Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import unittest
+
+import tensorflow as tf
+
+from elasticdl.proto import elasticdl_pb2
+from elasticdl.python.common.args import parse_worker_args
+from elasticdl.python.worker.worker import Worker
+from elasticdl_client.common.constants import DistributionStrategy
+
+
+class WorkerPSInteractionTest(unittest.TestCase):
```

Comment:
It seems that there is no PS created in `WorkerPSInteractionTest`

Reply:
Yes, we should rename it.